acquire/bearer.go

```diff
@@ -0,0 +1,130 @@
+package acquire
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/goph/emperror"
+)
+
+//TokenParser defines the function signature of a bearer token extractor from a payload
+type TokenParser func([]byte) (string, error)
+
+//ParseExpiration defines the function signature of a bearer token expiration date extractor
+type ParseExpiration func([]byte) (time.Time, error)
```

Comment:
Not sure why this type line and some other function signatures in this file aren't getting colored correctly? I can't find any typos/potential parsing issues.